include/arch/riscv/arch/32/mode/machine.h

```diff
@@ -9,19 +9,23 @@
 #include <util.h>
 #include <arch/model/smp.h>
 #include <stdint.h>
+#include <plat/machine/devices_gen.h>
+#include <plat/platform_gen.h>
+
+#define CLINT_MTIME_LO_OFFSET 0xbff8
+#define CLINT_MTIME_HI_OFFSET 0xbffc
```

Comment:
Shouldn't this use `CLINT_MTIME_OFFSET` for the low offset and `CLINT_MTIME_OFFSET + 4` for the high offset? (For little-endian riscv platforms, no idea if there are any big-endian ones.)

Reply:
Yes that's right but when I had it that way initially, the `FPU0001` test hung. Which is weird, maybe I did something else wrong. Reading the timestamp into a `uint64_t` is simpler though so I've just done.